x/posts/internal/types/msgs_test.go

```diff
@@ -179,9 +179,9 @@ func TestMsgLikePost_ValidateBasic(t *testing.T) {
 		error sdk.Error
 	}{
 		{
-			name:  "Invalid post id returns error",
+			name:  "Invalid post newID returns error",
```

Comment:
I think this is a refactoring error

```suggestion
			name:  "Invalid post id returns error",
```